physionet-django/notification/utility.py

```diff
@@ -509,8 +510,22 @@ def contact_reference(request, application, send=True, wordwrap=True,
         body = defaultfilters.wordwrap(body, 70)
 
     if send:
-        send_mail(subject, body, settings.DEFAULT_FROM_EMAIL,
-                  [application.reference_email], fail_silently=False)
+        if cc:
+            message = EmailMessage(
+                subject=subject,
+                body=body,
+                from_email=settings.DEFAULT_FROM_EMAIL,
+                to=[application.reference_email],
+                cc=[cc]
```

Comment:
Instead of the `if cc` clause, could you instead just pass the cc (which in some cases may be `None`)?

Reply:
I can investigate what happens if we try to pass a `None` as the cc

Reply:
Looks like it's `__init__` is `None` so it should work but I'll double-check with some tests:

https://github.com/django/django/blob/59e503b6708d41a44f2aa320272de3e2ecb5d65c/django/core/mail/message.py#L188-L196

Reply:
Also they initialize subject and body as `''` 🤔